packages/runtime/src/useCases/common/PasswordProtectionMapper.ts

```diff
@@ -3,6 +3,8 @@ import { PasswordProtectionDTO } from "@nmshd/runtime-types";
 import { PasswordProtection, PasswordProtectionCreationParameters } from "@nmshd/transport";
 
 export class PasswordProtectionMapper {
+    public static toPasswordProtectionDTO(passwordProtection: PasswordProtection): PasswordProtectionDTO;
+    public static toPasswordProtectionDTO(passwordProtection?: PasswordProtection): PasswordProtectionDTO | undefined;
```

Comment:
This definition is exactly the same as the one below. Can't you remove it? (I'm not very familiar with the typescript way of overloading)